leftwm-core/src/handlers/window_handler.rs

```diff
@@ -436,15 +436,12 @@ pub fn scratchpad_xyhw(xyhw: &Xyhw, scratch_pad: &ScratchPad) -> Xyhw {
     .into()
 }
 
-fn sane_dimension(config_value: Option<Size>, default_percent: f32, max_pixel: i32) -> i32 {
+fn sane_dimension(config_value: Option<Size>, default_ratio: f32, max_pixel: i32) -> i32 {
     match config_value {
-        Some(size) => match size {
-            Size::Percentage(percentage) if (0.0..0.9).contains(&percentage) => {
-                size.into_absolute(100.0) as i32 * max_pixel / 100
-            }
-            Size::Pixel(pixel) if (0..(max_pixel as f32 * 0.9) as i32).contains(&pixel) => pixel,
-            _ => (default_percent * max_pixel as f32) as i32,
-        },
-        _ => (default_percent * max_pixel as f32) as i32,
+        Some(size @ Size::Ratio(percentage)) if (0.0..=1.0).contains(&percentage) => {
```

Comment:
oops, forgot to rename that one

Reply:
If you mean the percentage we can change it in another pr as a passing thing.

Reply:
thanks ^_^